config/zones/ES-CN-FVLZ.yaml

```diff
@@ -196,6 +202,10 @@ parsers:
   consumption: ES.fetch_consumption
   production: ES.fetch_production
 sources:
+  EU-ETS, REE:
+    link:
+      https://drive.google.com/file/d/15UeUHyhcjVw8fGLc0Zx_HELSx20cw_-e/view?usp=share_link,
+      https://demanda.ree.es
```

Comment:
This seems to be the same link as for ENTSO-E, this seems like a misstake? Did you forget to update it?

Reply:
The source needs to be a link to the actual calculation so they can be verified by anyone at a later date.

It can be a spreadsheet or Jupyter notebook or something similar but it needs to have some sort of reproducibility.

Reply:
Ok... I can make a notebook with the calculations... Where can I upload it though? I assume you would want to have edit access, etc...?

Reply:
https://colab.research.google.com would be a good place to upload it, we don't need write access just read access so we can copy it if need be.